src/time/parse.gleam

```diff
@@ -0,0 +1,34 @@
+import gleam/int
+import gleam/string
+
+pub type LocalDateTime =
+  #(#(Int, Int, Int), #(Int, Int, Int))
+
+pub external fn parse_iso8601(datestring: String) -> LocalDateTime =
+  "iso8601" "parse"
+
+pub external fn to_gregorian_seconds(time: LocalDateTime) -> Int =
+  "packages_ffi" "to_gregorian_seconds"
```

Comment:
```suggestion
  "calendar" "datetime_to_gregorian_seconds"
```

Reply:
Done and removed the `.erl` file as it's no longer needed